### ISSUE TITLE
How to declare negative class for image classification

### ISSUE THREAD

**Issue by onmyway133** [apple/turicreate#581]
I've been playing with image classification with Turi and it works great. I also played with Watson Visual Recognition and they support negative class, which is a place we put all negative result (like not-hotdog). I think Watson Visual Recognition uses the same Transfer learning approach https://apple.github.io/turicreate/docs/userguide/image_classifier/how-it-works.html as Turi.

How can we declare negative class to put like all not-hotdog images? Or we can do it other ways? Thanks

Sorry for naive question, but my concern comes from https://medium.com/@timanglade/how-hbos-silicon-valley-built-not-hotdog-with-mobile-tensorflow-keras-react-native-ef03260747f3

> First The nature of our problem meant a strong imbalance in training data: there are many more examples of things that are not hotdogs, than things that are hotdogs. In practice this means that if you train your algorithm on 3 hotdog images and 97 non-hotdog images, and it recognizes 0% of the former but 100% of the latter, it will still score 97% accuracy by default!

**Comment by srikris** [comment]
You can use an explicit not-label class in your data to train on images. You can use the class weights feature to balance out the classes. But like the article said, you have to make sure you aren't measuring accuracy and are measuring precision-per class on the class of interest (e.g. hotdog)

**Comment by onmyway133** [comment]
@srikris thanks for the answer, can you please show me how to use "class weights" with Turi?
